### DISCUSSION TITLE
How to add Winetricks to Whisky and use it, inside the application?

### DISCUSSION THREAD

**Discussion by ferreira-ff** [Whisky-App/Whisky#292] [Q&A] [answered]
Hello. Can someone explain to me, the proper way to install Winetricks on Whisky, without brew? I just wanted to download the current version, extract it to the **Whisky’s** «**~/Library/Application Support/**» **folder** and call it, inside the _Whisky.app_.

Is there a way to incorporate it, as on WineSkin?

Any help would be appreciated. Kind regards.

**Comment by IsaacMarovitz** [comment, marked as answer]
Whisky checks if Winetricks is installed in your homebrew directory. Either use brew, or install it to that directory manually.

**Reply by enicck** [reply]
Do i install winetricks with the arm version of homebrew, or with the amd64 version?

**Reply by IsaacMarovitz** [reply]
New versions of Whisky come with Winetricks bundled, you don't need to manually install it

**Reply by enicck** [reply]
great!
but when I try to run a wintricks command via whisky I get a `no such file or directory` :(
